### ISSUE TITLE
Can't use share screen with meet on MacOS

### ISSUE THREAD

**Issue by ssbarnea** [getstation/desktop-app#95]
It seems that this app shares the same problem that cripples meet on other similar solutions like Ferdi, Frank or Rambox, the inability to share current screen in meet.

**Comment by jijojosephk** [comment]
@ssbarnea we had fixed in [teams-for-linux](https://github.com/IsmaelMartinez/teams-for-linux). Probably the same fix for this one as well. I'm not using Mac but someone else could take a look at the fix and cherry pick if that works.

**Comment by ssbarnea** [comment]
In fact there is an open pull-request on Rambox project opened since august that is supposed to fix the problem but nobody bothered to review it... I was about to switch from Rambox to Station for that reason just to discover i will still be unable to share the screen.

**Comment by viktor44** [comment]
https://meet.google.com/ doesn't work now because of outdated Electron
This bug should be retested after Electron upgrade.

**Comment by viktor44** [comment]
Both apps don't work in Station now.
So this issue is blocked by #174 #80

**Comment by Mathijs003** [comment]
Having the same issue, any updates?

**Comment by supernoi** [comment]
pop-os 22.04
station 3.10

Same problem when trying to share the screen in Teams:

"Your video isn't working
We couldn't access your camera."

**Comment by gabimig** [comment]
I see that my original Issue was closed and merge into this one by @viktor44 , This issue have been open since 2021 I would like to know if it is planed a fix, or screen sharing is a feature that won't have support?

**Comment by viktor44** [comment]
this feature is still waiting for its hero.
PRs are welkomed :)
